rules/chaos.go

```diff
@@ -0,0 +1,32 @@
+package rules
+
+import (
+	"k8s.io/client-go/pkg/api/v1"
+	"math/rand"
+	"os"
+	"strconv"
+	"fmt"
+)
+
+const ENV_CHAOS_CHANCE = "CHAOS_CHANCE"
+
+type chaos struct {
+	chance float64
+}
+
+func (rule *chaos) load() (bool, error) {
+	value, active := os.LookupEnv(ENV_CHAOS_CHANCE)
+	if !active {
+		return false, nil
+	}
+	chance, err := strconv.ParseFloat(value, 64)
+	if err != nil {
+		return false, fmt.Errorf("invalid chaos chance %s", err)
+	}
+	rule.chance = chance
+	return true, nil
+}
+
+func (rule *chaos) ShouldReap(pod v1.Pod) (bool, string) {
+	return rand.Float64() < rule.chance, "was flagged for chaos"
```

Comment:
We should probably set the seed to something given that by default the Seed will revert to a value of 1.
https://golang.org/pkg/math/rand/#Seed

Something like `time.Now().UnixNano()` might by simple enough for now.